public/app/features/search/components/SearchCard.tsx

```diff
@@ -18,15 +18,40 @@ export interface Props {
   onToggleChecked?: OnToggleChecked;
 }
 
-export function getThumbnailURL(uid: string, isLight?: boolean) {
-  return `/api/dashboards/uid/${uid}/img/thumb/${isLight ? 'light' : 'dark'}`;
-}
+export const getThumbnailURL = (uid: string, isLight?: boolean) =>
```

Comment:
I know this is just a style thing... but what is wrong with exporting simple functions!

Reply:
haha I just touched the method and my IDE autoconverted it - I'll disable that